packages/mobile/src/screens/profile-screen/ProfileTabs/CollectiblesTab.tsx

```diff
@@ -175,7 +182,7 @@ export const CollectiblesTab = () => {
       renderItem={({ item }) => (
         <CollectiblesCard
           collectible={item}
-          ownerId={profile.user_id}
+          ownerId={user_id}
```

Comment:
possible that this renders empty first b/c user_id is null/undefined?

Reply:
great catch, just updated